buildSrc/src/main/java/Dependencies.kt

```diff
@@ -15,7 +15,7 @@ object Versions {
     const val androidx_appcompat = "1.2.0-rc01"
     const val androidx_biometric = "1.1.0-alpha01"
     const val androidx_coordinator_layout = "1.1.0-rc01"
-    const val androidx_constraint_layout = "2.0.0-beta6"
```

Comment:
I'm guessing there was a reason that we were using the beta version here. Can you look for the blame of that and double check that it's ok to downgrade to the stable version?

Reply:
Oops, I didn't mean to put this change in this PR